stubs/xmltodict/METADATA.toml

```
@@ -1,2 +1,2 @@
-version = "1.0.*"
+version = "1.0.1"
```

```suggestion
version = "~= 1.0.1"
```

This way we don't need to update the stubs for each minor version.